### PR TITLE
Fixing and issue that would generate non sense value when referring a numeric value in a  string

### LINKED ISSUE #6: referencing numbers breaks referencing value
basically we found this case of:

```
port: 3000
url: domain.com:{{port}}
```

and `config.get('url')` would only return `3000`. Weird enough but we should have a test and look at it.

### PR DESCRIPTION
**Pull request by unlucio**
fixes #6

### CODE REVIEW

File: test/reconfig.js [hunk 1]
```diff
@@ -112,6 +112,18 @@ describe('Reconfig', function() {
       }));
     });
 
+    it('should go nuts with numeric self-referencing configs', function() {
```
Comment (odino) [inline test/reconfig.js]: What? :)

Comment (unlucio) [inline test/reconfig.js]: maybe shouldn't? O.o

Comment (odino) [inline test/reconfig.js]: :)

File: test/reconfig.js [hunk 2]
```diff
@@ -112,6 +112,18 @@ describe('Reconfig', function() {
       }));
     });
 
+    it('should not go nuts with numeric value for self-referencing configs', function() {
```
Comment (unlucio) [inline test/reconfig.js]: any better?

**Comment by odino** [comment]
`v1.6.2` :)

**Pull request merged**